src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,44 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private static final int STACK_CAPACITY = 12;
+    private static final int EMPTY_STACK_VALUE = -1;
+    private static final int STACK_FULL_VALUE = 11;
+
+    private final int[] NUMBERS = new int[STACK_CAPACITY];
+    private int total = EMPTY_STACK_VALUE;

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)